### PR TITLE
UI/calendar

### PR DESCRIPTION
**Pull request by pulimoodan**
Tried to implement the exact UI design as it's shown in the sample image.

There are some breaking changes in calendar UI:

- Separated calendar header and month navigator components
- Created active day and hour cursor components
- Created event card components
- Adds one more variant to the buttons

Screenshot (25)

Of course, we need to refactor the components for better reusability.

### CODE REVIEW

**Comment by pulimoodan** [comment]
Related to: #10

**Comment by anuragts** [comment]
can you escape ' quotes 
image

File: src/app/(application)/calendar/components/EventCard.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+function EventCard({ variant }: Props) {
+  return (
+    <div
+      className={`w-[100%] h-[100%] border border-${variant}-500 rounded-xl p-[2px] bg-${variant}-100/50`}
+    >
+      <div
+        className={`w-[100%] h-[100%] border border-${variant}-500 rounded-[10px] p-2 flex flex-col`}
+      >
+        <div className="flex-1">
+          <h4 className={`font-bold text-${variant}-700`}>Carl's birthday</h4>
```
Comment (anuragts) [inline src/app/(application)/calendar/components/EventCard.tsx]: just change this to Carl BirthDay

File: src/app/(application)/calendar/components/EventCard.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { IconDots } from "@tabler/icons-react";
+import React from "react";
+
+interface Props {
+  variant: "green" | "red" | "gray" | "orange" | "yellow" | "blue";
+}
+
+function EventCard({ variant }: Props) {
```
Comment (anuragts) [inline src/app/(application)/calendar/components/EventCard.tsx]: As this is a event card component , can you pass all the stuff like title , time etc as  props. This is not necessary if you want you can do it , or else I'll merge this PR

**Review by anuragts — CHANGES_REQUESTED** [review]
Changes you implement these changes

File: src/app/(application)/calendar/components/EventCard.tsx (hunk 2 above)
Comment (pulimoodan) [reply src/app/(application)/calendar/components/EventCard.tsx]: I can do that. I thought those things would be done later.

File: src/app/(application)/calendar/components/EventCard.tsx (hunk 1 above)
Comment (pulimoodan) [reply src/app/(application)/calendar/components/EventCard.tsx]: Sure

**Review by anuragts — COMMENTED** [review]
LGTM

**Comment by anuragts** [comment]
can you resolve the merge conflits  @pulimoodan , just accept the main branch changes of CreateEvent.tsx

**Pull request merged**